src/unix/os390.c

```diff
@@ -199,7 +200,12 @@ uint64_t uv_get_total_memory(void) {
 
 
 uint64_t uv_get_constrained_memory(void) {
-  return 0;  /* Memory constraints are unknown. */
+  struct rlimit rl;
+
+  if (getrlimit(RLIMIT_MEMLIMIT, &rl) == 0)
+    return rl.rlim_cur;
```

Comment:
https://www.ibm.com/support/knowledgecenter/SSLTBW_2.2.0/com.ibm.zos.v2r2.bpxbd00/rgrlmt.htm says that `RLIMIT_MEMLIMIT` returns a value "in 1 megabyte segments". The API documentation for `uv_get_constrained_memory()` currently says that it returns bytes:
https://github.com/libuv/libuv/blob/v1.x/docs/src/misc.rst
```
.. c:function:: uint64_t uv_get_constrained_memory(void)

    Gets the amount of memory available to the process (in bytes) based on
    limits imposed by the OS. If there is no such constraint, or the constraint
    is unknown, `0` is returned. Note that it is not unusual for this value to
    be less than or greater than :c:func:`uv_get_total_memory`.

    .. note::
        This function currently only returns a non-zero value on Linux, based
        on cgroups if it is present.

    .. versionadded:: 1.29.0
```

Reply:
Added conversion to bytes and updated documentation.